shared/common-adapters/search-filter.tsx

```diff
@@ -224,13 +224,14 @@ class SearchFilter extends React.PureComponent<Props, State> {
     }
     if (Styles.isMobile) {
       return (
-        <Kb.Icon
-          type="iconfont-remove"
-          sizeType={this._iconSizeType()}
-          onClick={this.props.mobileCancelButton ? this._clear : this._cancel}
-          color={this._iconColor()}
-          style={styles.removeIconNonFullWidth}
-        />
+        <Kb.ClickableBox onClick={this.props.mobileCancelButton ? this._clear : this._cancel}>
```

Comment:
i believe you can just do this with padding, but this is ok also

Reply:
Didn't seem to work, but maybe I'm doing something wrong, adding padding to the icon still gives:

<img width="38" alt="Screen Shot 2019-09-06 at 11 53 33 AM" src="https://user-images.githubusercontent.com/21217/64453260-fd05c480-d09c-11e9-81f3-2ca0f93e7462.png">

Reply:
Did add padding to the original or on top of this version. Likely only works with an onclick

Reply:
to original